src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Fixing `Remove-AzVmssDataDisk` bug. Now the cmdlet will work when there is just one data disk attached to a Vmss. 
```

Comment:
Suggested to "Fixed a bug when 1 data disk attached to VMSS for `Remove-AzVmssDataDisk` [#13368]"